### PR TITLE
fix failing failover testing

### PR DESCRIPTION
**Pull request by huone1**
Signed-off-by: huone1 <huwanxing@huawei.com>

**What type of PR is this?**
/kind flake

**What this PR does / why we need it**:
fix the failover e2e test failed
the rescheduling 
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/rescheduling_test.go [hunk 1]
```diff
@@ -45,67 +63,58 @@ var _ = ginkgo.Describe("reschedule testing", func() {
...
+					},
+					ClusterNamespace:  "karmada-cluster",
+					ClusterName:       newClusterName,
+					ClusterContext:    clusterContext,
+					ClusterKubeConfig: kubeConfigPath,
+				}
+				err := karmadactl.RunJoin(os.Stdout, karmadaConfig, opts)
+				gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+
+				// wait for the current cluster status changing to false
```
Comment (XiShanYongYe-Chang) [inline test/e2e/rescheduling_test.go:87]: // wait for the current cluster status changing to true

**Review by RainbowMango — APPROVED** [review]
/approve

File: test/e2e/rescheduling_test.go (hunk 1 above)
Comment (huone1) [reply test/e2e/rescheduling_test.go]: it is fixed

**Pull request merged**